Marlin/src/lcd/e3v2/creality/dwin_lcd.cpp

```diff
@@ -127,8 +127,8 @@ void DWIN_Startup(void) {
   delay(750);   // Delay here or init later in the boot process
   const bool success = DWIN_Handshake();
   if (success) DEBUG_ECHOLNPGM("ok."); else DEBUG_ECHOLNPGM("error.");
-  DWIN_Frame_SetDir(TERN(DWIN_MARLINUI_LANDSCAPE, 0, 1));
-  #if IS_DWIN_MARLINUI || DISABLED(SHOW_BOOTSCREEN)
+  DWIN_Frame_SetDir(1);
```

Comment:
Is the orientation set anywhere else now? If not, then landscape mode is no longer supported.

Reply:
I don't recognize this regression. If the orientation is not being set in the handshake or DWIN_init, or something like that, then this change should be reverted.